tests/functional/karate_tests/README.md

```diff
@@ -0,0 +1,49 @@
+# openwhisk-karate
+This repository contains all the Test Cases needed to do various forms of automated test on the OW environments.These tests will also complement the existing Scala based Test cases.
+These tests are based on the BDD model and enabled quick on-boarding of the tests.
+It is based of karate (https://github.com/intuit/karate) framework. 
+
+
+### How to run functional test
+1. Clone/Download the repo
+2. Navigate to the root folder
+2. Use the following command to run the above selected suite: `mvn test -Dtest="com.karate.openwhisk.smoketests.SmokeTestRunner"` (This will run all the tests in com.karate.openwhisk.smoketests package.)
+
+### How to run the performance test 
+1. Navigate to the root path.Check out the performance test package (/ow-karate/src/test/java/com/karate/openwhisk/performance)
+2. setUp(createActionTest.inject(rampUsers(5) over (5 seconds))
+    ).maxDuration(1 minutes).assertions(global.responseTime.mean.lt(1100))
+Tweak the above values as per your test needs/load requirements
+3. Use the following command to run the performance: 
+`mvn clean install gatling:test -Dgatling.simulationClass=com.karate.openwhisk.performance.LoadTest`
```

Comment:
I don't think these instructions are clear - I think you need to edit a file, but it is unclear which file or why?
Please provide a gradle command